### ISSUE TITLE
Roborock pin for Roborock Q5 Pro+ 

### ISSUE THREAD

**Issue by ArthFink** [AlexxIT/go2rtc#1521]
Hey, 

I know that the Q5 Pro+ is not “officially” tested, but I got the link for my Roborock via go2rtc.

Now I am trying to find this pin, where this pin usually is found? I´ve looked all over the vacuum and can't something that looks like the right pin..

**Comment by AlexxIT** [comment]
Pin is set by user manually. You needs to enter it each time you open stream in official mobile app.

**Comment by ArthFink** [comment]
ah okay thx for the quick response. I don't have the functionality in my app. 
But I got a link in go2rtc, can you explain what this means? Is i possible that Roborock just didn't enable the feature for the Q5 Pro+ or is there another expiation?

**Comment by AlexxIT** [comment]
Do you have camera in your vacuum?

**Comment by ArthFink** [comment]
I did not find a camera mentioned in the Manuel, I think not. 
I had just followed a tutorial, and they had used go2rtc, now I understood it's not needed for my use case.

Now I'm just interested what this is? Why I have a link if I don't have a camera, or does this link just link to my user account and not the device itself? 
![5427354343377594388](https://github.com/user-attachments/assets/d6c855c7-e02f-41b2-9cc4-c115a29574bb)
